### PR TITLE
Dev - Bug Bash Fixes

### PR DESCRIPTION
**Pull request by essiequoi**
# What does this PR do?

Merges some changes from the bug bash and recently added docs

### CODE REVIEW

**Comment by essiequoi** [comment]
@robert-s-lee the best way to review this will be to checkout the branch and preview the site with `yarn start`

**Review by robert-s-lee — COMMENTED** [review]
0.8.17 and 0.8.16 have the same description.

**Review by robert-s-lee — COMMENTED** [review]
free as in "free beer"! -- what guarantees is Grid granting with this statement?

File: docs/features/datastores/1_README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+download, extract, or otherwise "process" before you can move on to the real work. 
+
+:::
+
+We know how important a role your data plays in everything you run on Grid. We've put an
+incredible amount of time and effort into creating a truly unique *optimization pipeline*
+which removes every bit of latency possible from the point your program calls `with
+open(filename, 'r') as f:` to the instant that data is provided to you. You'll find
+traversing the data directory structure in a [session](../../features/sessions/README.md)
+indistinguishable from the experience of `cd`-ing around your local workstation. File
```
Comment (robert-s-lee) [inline docs/features/datastores/1_README.md:50]: how accruate is this description

File: docs/features/datastores/2_Using Datastores/1_README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+sidebar_label: Using Datastores
+---
+# How to use Datastores
+
+There are many ways to operationalize a Datastore in Grid. Below we provide some examples
+to get you started with working with this core feature. The examples cover the following:
+
+1. [Creating Datastores](./2_creating-datastores.md)
+   
+    1.1. Datastores from your machine
```
Comment (robert-s-lee) [inline docs/features/datastores/2_Using Datastores/1_README.md:12]: 1 level 1.x sub levels, but other sections do not.  any reason for this?

File: docs/features/datastores/2_Using Datastores/2_creating-datastores.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
+---
+title: Creating Datastores
+sidebar_label: Creating Datastores
+---
+# Create Datastores
+
+Datastores can be created from a number of sources (on both the CLI and UI):
+
+- Uploading files stored on your computer
+- From an HTTP URL which downloads a `.zip` or `.tar.gz` file. These files are
```
Comment (robert-s-lee) [inline docs/features/datastores/2_Using Datastores/2_creating-datastores.md:10]: s/download/upload/

File: docs/features/datastores/2_Using Datastores/2_creating-datastores.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+```bash
+pip install lightning-grid --upgrade
+grid login
+```
+
+Next, use the Datastores command to upload any folder:
+
+```bash
+grid datastore create ./imagenet_folder/ --name imagenet
```
Comment (robert-s-lee) [inline docs/features/datastores/2_Using Datastores/2_creating-datastores.md:54]: although this works, dose not follow the CLI convention 

s/
grid datastore create ./imagenet_folder/ --name imagenet/
grid datastore create --name imagenet ./imagenet_folder/ 
/

File: docs/features/datastores/1_README.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
+---
```
Comment (robert-s-lee) [inline docs/features/datastores/1_README.md:1]: what is the URL to access this file?  looks like [2_Using Datastores/1_README.md] is overwriting this page.

File: docs/features/datastores/1_README.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+In order to ensure complete data privacy & flexibility of use, Grid never attempts to
+process the contents of the files or infer/optimize for any particular usage behaviors based
+on file contents. 
+
+:::
+
+## Why use Datastores
+
+Datastores are zero-cost datasets (yes, we really do mean that they are free too you -
```
Comment (robert-s-lee) [inline docs/features/datastores/1_README.md:27]: zero-cost only for the community.  BYOC customers will get charged by AWS

File: docs/features/datastores/2_Using Datastores/4_attaching-datastores.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+On your local machine, you call the script like this:
+
+```bash
+python main.py --root my_dataset/
+```
+
+Your script uses the dataset like this:
+
+```python
+args.add_argument('-root')
```
Comment (robert-s-lee) [inline docs/features/datastores/2_Using Datastores/4_attaching-datastores.md:64]: s/-root/--root/

**Review by robert-s-lee — APPROVED** [review]
Fixed broken links.  other content improvements will be left for later releases.

**Pull request merged**